test/integration/default/controls/ard_test.rb

```diff
@@ -1,23 +1,31 @@
 title 'remote access'
 
+user_has_access  = 1 << 31
+text_messages    = 1 << 0
+control_observe  = 1 << 1
+send_files       = 1 << 2
+delete_files     = 1 << 3
+generate_reports = 1 << 4
+open_quit_apps   = 1 << 5
+change_settings  = 1 << 6
+restart_shutdown = 1 << 7
+show_observe     = 1 << 30
+
+priv = text_messages | control_observe | send_files | delete_files | generate_reports | open_quit_apps | change_settings | restart_shutdown | show_observe
+
 control 'remote-control' do
   title 'naprivs value represents remote control for all users'
-  desc 'Verify that naprivs has the magic value to indicate all privileges'
+  desc "verify that naprivs has the bitmask value #{priv}"
 
-  # user_has_access  = 0b1 << 31
-  # text_messages    = 0b1 << 0
-  # control_observe  = 0b1 << 1
-  # send_files       = 0b1 << 2
-  # delete_files     = 0b1 << 3
-  # generate_reports = 0b1 << 4
-  # open_quit_apps   = 0b1 << 5
-  # change_settings  = 0b1 << 6
-  # restart_shutDown = 0b1 << 7
-  # observe_only     = 0b1 << 8
-  # show_observe     = 0b1 << 30
+  describe command("/usr/bin/dscl . list /Users naprivs | tr -ds 'vagrant-' '[:space:]'") do
+    its('stdout.to_i') { should match user_has_access }
+  end
+
+  describe command('defaults read /Library/Preferences/com.apple.RemoteManagement ARD_AllLocalUsersPrivs') do
+    its('stdout.to_i') { should match priv }
+  end
 
   describe command('/usr/libexec/PlistBuddy -c Print /Library/Preferences/com.apple.RemoteManagement.plist') do
```

Comment:
This kind of conflicts with my other comment using `defaults`, but it might look nice here to combine the plist reads into one call and check for both values:

```
  describe command('/usr/libexec/PlistBuddy -c Print /Library/Preferences/com.apple.RemoteManagement.plist') do
    its('stdout') { should match 'ARD_AllLocalUsers = true' }
    its('stdout') { should match "ARD_AllLocalUsersPrivs = #{all_privileges}" }
  end
```